src/realm/column_type_traits.hpp

```diff
@@ -33,7 +35,6 @@ class Decimal128;
 class ObjectId;
```

Comment:
can this be removed as well since it is now fully included?

Reply:
Yes